scripts/diagnose_knob.py

```
@@ -0,0 +1,116 @@
+BRIDGE_IP = sys.argv[1] if len(sys.argv) > 1 else "192.168.50.225"
+ZONE_ID = sys.argv[2] if len(sys.argv) > 2 else ""
+BRIDGE_PORT = 8088
+UDP_PORT = BRIDGE_PORT + 1
+MAGIC = 0x524B
+
+def fetch_json(path):
+    url = f"http://{BRIDGE_IP}:{BRIDGE_PORT}{path}"
+    try:
+        with urllib.request.urlopen(url, timeout=5) as r:
+            return json.loads(r.read())
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)